src/app/shipping/MultiShippingForm.spec.tsx

```diff
@@ -92,5 +96,23 @@ describe('MultiShippingForm Component', () => {
                 })
             );
         });
+
+        it('renders address form modal when shoppers choose new address', async () => {
+            component.find('.consignmentList > li')
```

Comment:
Prefer to use `data-test` to select elements in tests rather than CSS class names.